src/Communication/docs/Set-AzCommunicationServiceNotificationHub.md

```diff
@@ -0,0 +1,207 @@
+---
+external help file:
+Module Name: Az.Communication
+online version: https://docs.microsoft.com/en-us/powershell/module/az.communication/set-azcommunicationservicenotificationhub
+schema: 2.0.0
+---
+
+# Set-AzCommunicationServiceNotificationHub
+
+## SYNOPSIS
+Links an Azure Notification Hub to this communication service.
+
+## SYNTAX
+
+### LinkExpanded (Default)
+```
+Set-AzCommunicationServiceNotificationHub -CommunicationServiceName <String> -ResourceGroupName <String>
+ -ConnectionString <String> -ResourceId <String> [-SubscriptionId <String>] [-DefaultProfile <PSObject>]
```

Comment:
>ResourceId [](start = 29, length = 10)

Probably want to specify that this is the notification hub resource ID. Otherwise it's ambiguous.

Reply:
Done